FrameworkSystem/Client/WebAppCompiler.py

```diff
@@ -0,0 +1,353 @@
+"""
+It is used to compile the web framework
+"""
+
+
+__RCSID__ = "$Id$"
+
+import os, tempfile, shutil, subprocess, gzip, sys, imp
+
+from DIRAC import gLogger, gConfig, rootPath, S_OK, S_ERROR
+from DIRAC.Core.Utilities.CFG import CFG
+
+class WebAppCompiler( object ):
+  
+  def __init__( self, params ):
+    
+    self.__params = params
+    
+    self.__extVersion = '4.2.1.883'
```

Comment:
what's this, and why hardcoded version?

Reply:
This is for backward compatibility. This is used by the compilation template and it is the version of Extjs library. In some point I would like to remove it (This is a "mistake" in the current procedure, I must not use extjs version). I will replace:
 <link rel="stylesheet" type="text/css" href="/DIRAC/static/extjs/%EXT_VERSION%/resources/css/ext-all.css" />
to 
 <link rel="stylesheet" type="text/css" href="/DIRAC/static/extjs/resources/css/ext-all.css" />

In principle this is depend on you and Andrei. So, if you create the DIRAC tarball using this new procedure, I can drop it...